module-example-2/src/model.ts

```diff
@@ -0,0 +1,16 @@
+import { objectType } from 'nexus/dist';
+
+export const CoreUnit = objectType({
+  name: 'CoreUnit',
```

Comment:
Why redefining the same type here doesn't break typescript or nexus?

Reply:
it is not exported from the `api/modules` (look up index.ts) changes. So there is no conflict since only one place (the module-example directory) has this exported.

Reply:
But would it throw an error if it exports the same type?

Reply:
if on [ae7d149](https://github.com/powerhouse-inc/switchboard-boilerplate/pull/67/commits/ae7d14910407fba42b1dc977778a459e07c02996) i change the name of the type from `Counter` to `CoreUnit` - no error is thrown and it seems like it just overwrites the logic there.

Although due to redefinition above the typecheck command throws an error for a resolver for the actual core unit. So seems like the typesystem in the CI is still able to catch the conflicting behaviour.